hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/router/RouterRpcServer.java

```diff
@@ -284,6 +304,12 @@ public RouterRpcServer(Configuration conf, Router router,
     int handlerQueueSize = this.conf.getInt(DFS_ROUTER_HANDLER_QUEUE_SIZE_KEY,
         DFS_ROUTER_HANDLER_QUEUE_SIZE_DEFAULT);
 
+    this.enableAsync = conf.getBoolean(DFS_ROUTER_RPC_ENABLE_ASYNC,
+        true);
```

Comment:
@KeeProMise Sir, a minor opinion. use default value here.

Reply:
@hfutatzhanghb Thank you for your careful review, done!